app/assets/javascripts/models/scorecard.js

```diff
@@ -6,6 +6,7 @@ App.Scorecard = DS.Model.extend({
   user: DS.belongsTo('user', { async: true }),
   userInitials: DS.attr('string'),
   turns: DS.hasMany('turn', { async: true }),
+  isStarted: DS.attr('boolean'),
```

Comment:
I think this needs to be a computed property, otherwise, the isStarted will only update on a complete app refresh (or an additional call to the DB, which isn't made with rounds).

Can confirm: just tested it out and played about 6 holes:

![screen shot 2015-01-21 at 7 08 20 pm](https://cloud.githubusercontent.com/assets/2623668/5847930/e3d40b00-a1a0-11e4-9495-4b6783d3a81b.png)